colab_tutorial/Fraud_Detection_Example.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
Should change this directory to notebooks/

Reply:
+1 Sterling's comment.